### DISCUSSION TITLE
Question about Fiber Connections

### DISCUSSION THREAD

**Discussion by kotso** [netbox-community/netbox#7966] [Q&A] [answered]
We have started filling up our fiber infrastructure into the netbox and came up with question, what is correct way to do it:

1. We have multiple location (5) and fiber cables between them, ODF installed on each location and everything is interconnected.
2. Final devices (interfaces) some are connected with simplex LC connectors, some with duplex LC connectors.

Currently As I found LC as connector type, I have added all ODF front and rear ports as LC, and have done correct matching between ODF-s. 

On Final ODF's where link goes from ODF to Device(interface) I have number of LC front connectors, and I need to connect it to Interface on device.

Issue is that some interfaces are connected with 2x cables, some with 1. 

I did not find type of connector LC Simplex and LC Duplex to manage it accordingly.

So what would be correct way to do this?

**Comment by candlerb** [comment, marked as answer]
Netbox doesn't model this well.

Your best option for now is to connect the interface to one port of the ODF, and set "mark connected" on the adjacent port on the ODF.

![image](https://user-images.githubusercontent.com/44789/144404163-fbcdeba8-d707-4793-9377-f4bab098f41a.png)

**Comment by kotso** [comment]
ok, sounds like reasonable workaround, thank you @candlerb

**Comment by zombah** [comment]
I just change name on ODF front ports like if duplex connect - pair1-2, and if simplex just - port3,
odf port count not exactly reflect reality but pretty fine to track.
